src/v3/src/components/Button/Button.tsx

```diff
@@ -87,7 +87,12 @@ const Button: UISchemaElementComponent<{
             // compensate the offset from the CircularProgress component
             marginRight="23px"
           >
-            <CircularProgress sx={{ color: 'white' }} />
+            <CircularProgress
+              // TODO: OKTA-518793 - replace english string with key once created
+              aria-label="Loading..."
+              aria-valuetext="Loading..."
```

Comment:
Was there a reason we can't put the new translations into the properties file now?

Reply:
Lee actually has that ticket and is currently working on it with Product and design. There is a process to get new translations added, mostly just need to agree on the Copy text and then it can be added to master, then the service should pick it up in the next release and generate the text in the other languages.